### ISSUE TITLE
[Welcome] Welcome pop-up lighting image has no alt text

### ISSUE THREAD

**Issue by Krishna1507** [veshu/accessibility-bughunt-bluesource#72]
**Describe the bug**
Welcome popup lighting image has no alt text

**Step to reproduce**
1) Go to app localhost:4200/home
2) Welcome message popup
3) Inspect element the welcome pop up lighting image 
4) Observer there is not alt text for that image

**Expected behavior**
The image should be the alt="<text>"  text, blank alt text ensure that it is ignored by a screen-reader and is hidden when images are disabled or unavailable.

**Device**: Labtop
**Operating System**: Windows 10
**Browser**: Firefox 70

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non text content
**Tools used**: WAVE

**Attachments**
https://drive.google.com/open?id=1Od_3Z1XKg-LNhZegeEkraBz0wUUQAEhB
